README-IT.md

```diff
@@ -3,12 +3,12 @@
 [![Stato costruzione](https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true)](https://ci.appveyor.com/project/CosmosOS/cosmos/branch/master)
```

Comment:
This file can be deleted, theres a traductor on the cosmos website.

Reply:
Can you also delete README-fr md please? I forgot to remove it before merging my pr